pom.xml

```diff
@@ -100,7 +96,6 @@
     <module>errai-javaee-all</module>
     <module>errai-config</module>
     <module>errai-codegen-gwt</module>
-    <module>errai-cordova</module>
```

Comment:
removed originally in https://github.com/errai/errai/commit/765202c4fd791116e5ed0006f33c357bb7f52783